ISISScientificComputing/autoreduce
# reduction variables not displayed

### Summary of work
* Ensure that Variables are initialised and saved correctly

### How to test your work
* Ensure this code is running on the dev compute node.
* Submit a run manually and ensure the variables are saved (either database or webapp inspection)

### Additional comments
Note: I am unable to test this fully as the isis data archive is down for the rest of today!! Please validate this tomorrow 

Fixes #619


**Before merging ensure the release notes have been updated**
